docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+
+```
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe create-instance --instance "Tentacle" --config "C:\Octopus\Tentacle.config"
+Tentacle.exe new-certificate --instance "Tentacle" --if-blank
+Tentacle.exe configure --instance "Tentacle" --reset-trust
+Tentacle.exe configure --instance "Tentacle" --app "E:\Octopus\Applications" --home "E:\Octopus\Home" --port "10933" --noListen "False"
+Tentacle.exe configure --instance "Tentacle" --trust "<OCTOPUS SERVER THUMBPRINT>"
+"netsh" advfirewall firewall add rule "name=Octopus Deploy Tentacle" dir=in action=allow protocol=TCP localport=10933
+Tentacle.exe service --instance "Tentacle" --install --stop --start
+```
+In the script block above, the following actions were performed:
+ - Installed the tentacle instance using the default instance name of "Tentacle" and make sure the Tentacle.config file was installed into the default location of "C:\Octopus\Tentacle.config". Since C:\ is local storage this file does not get swapped when we perform a swap to a new active node.
+ - Configured the new instance to listen on TCP Port 10933 then save the Application and Home data onto our Shared Storage drive which in this example is mounted as E:\
+ - Configured the tentacle to trust our Octopus Server holding a certificate which matches the Certificate Thumbprint
+ - Ensured the Windows Firewall has a rule configured to allow incoming connections on TCP Port 10933 which will allow the Octopus Server to talk to our new Tentacle.
+
+Using the Tentacle Manager stop the Octopus Tentacle service which was just installed on the first node and take the shared disk offline in Windows Disk Management. 
+
+Now go to the second tentacle server in the Active/Passive cluster and bring the same disk online, repeating the steps which were just performed on the first node to install the Octopus Tentacle service. Please keep the Octopus Tentacle service started and ensure that the Shared Storage is still mounted this time so that the SSL Certificate can be exported out of Octopus Tentacle.
+
+## Generate an Octopus Tentacle Private Key {#ClusteringTentacles-NewPrivateKey}
+
+Open `cmd` again and run these commands to generate a new Private Key from an Octopus Deploy Tentacle.
+```
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe new-certificate -e "TentacleClusterPrivateKey.txt"
+```
+
+*![](/docs/images/clustered-listening-tentacles/export-privatekey.jpg)*
+
+## Import the new Octopus Tentacle Private Key {#ClusteringTentacles-ImportPrivateKey}
+
+Now import the new private key into the server from which it was just generated.
+```
+Tentacle.exe import-certificate -instance "Tentacle" -f "TentacleClusterPrivateKey.txt"
```

Comment:
Will need password if we are exporting with a password
`pem`, not `txt`
full path to export
`--from-file` instead of `-f`